ProcessMaker/Traits/ProjectAssetTrait.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace ProcessMaker\Traits;
+
+trait ProjectAssetTrait
+{
+    public function assignAssetsToProjects($request, $assetModelClass)
+    {
+        if ($request->input('projects')) {
+            $projectAssetModelClass = 'ProcessMaker\Package\Projects\Models\ProjectAsset';
+            $projectAssets = new $projectAssetModelClass;
+            $projectIds = (array) $request->input('projects');
+            $assetData = [];
+
+            foreach ($projectIds as $id) {
+                $assetData[] = [
+                    'asset_id' => $this->id,
+                    'project_id' => $id,
+                    'asset_type' => $assetModelClass,
+                ];
+            }
+
+            $projectAssets::createMany($assetData);
```

Comment:
Not a big deal – but when this method is used, the `updated_at` and `created_at` columns of the assets table are not populated.

Reply:
Good catch!

Reply:
Observation ticket created:
https://processmaker.atlassian.net/browse/FOUR-10126

marking as resolved